src/components/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+
+interface IProps {
+  inverted?: boolean;
```

Comment:
this prop could be called `isInverted`, so we know from the sound of it, that we're dealing with a boolean.
It's a good naming convention.

Reply:
done